CMakeLists.txt

```diff
@@ -1,131 +1,163 @@
-cmake_minimum_required(VERSION 3.10 FATAL_ERROR)
-set(CMAKE_CXX_STANDARD 14)
-set(CMAKE_CXX_STANDARD_REQUIRED ON)
-set(CMAKE_CXX_EXTENSIONS OFF)
+cmake_minimum_required(VERSION 3.10 FATAL_ERROR) set(CMAKE_CXX_STANDARD 14) set(
+    CMAKE_CXX_STANDARD_REQUIRED ON) set(CMAKE_CXX_EXTENSIONS OFF)
 
-# Hide symbols by default
-set(CMAKE_CXX_VISIBILITY_PRESET hidden)
-set(CMAKE_VISIBILITY_INLINES_HIDDEN YES)
+#Hide symbols by default
+    set(CMAKE_CXX_VISIBILITY_PRESET hidden) set(
+        CMAKE_VISIBILITY_INLINES_HIDDEN YES)
 
-set(PROJECT_NAME neXtSIMutils)
-set(LIB_NAME domain_decomp)
-set(EXEC_NAME decomp)
-set(VERSION_MAJOR 1)
-set(VERSION_MINOR 0)
-project(${PROJECT_NAME} VERSION ${VERSION_MAJOR}.${VERSION_MINOR} LANGUAGES CXX)
+        set(PROJECT_NAME neXtSIMutils) set(LIB_NAME domain_decomp) set(
+            EXEC_NAME decomp) set(VERSION_MAJOR 1) set(VERSION_MINOR 0)
+            project(${PROJECT_NAME} VERSION ${VERSION_MAJOR}.${
+                VERSION_MINOR} LANGUAGES CXX)
 
-# Set RPATH behaviour
-set(CMAKE_SKIP_RPATH FALSE)
-set(CMAKE_SKIP_BUILD_RPATH FALSE)
-set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
-# Embed absolute paths to external libraries that are not part of
-# the project, (they are expected to be at the same location on all
-# machines the project will be deployed to
-set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE)
-include(GNUInstallDirs)
-# Define RPATH for executables via a relative expression to enable a
-# fully relocatable package
-file(RELATIVE_PATH relDir1
- ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}
- ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR})
-file(RELATIVE_PATH relDir2
- ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_BINDIR}/examples
- ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_INSTALL_LIBDIR})
-set(CMAKE_INSTALL_RPATH $ORIGIN/${relDir1}:$ORIGIN/${relDir2})
+#Set RPATH behaviour
+                set(CMAKE_SKIP_RPATH FALSE) set(
+                    CMAKE_SKIP_BUILD_RPATH
+                        FALSE) set(CMAKE_BUILD_WITH_INSTALL_RPATH FALSE)
+#Embed absolute paths to external libraries that are not part of
+#the project, (they are expected to be at the same location on all
+#machines the project will be deployed to
+                    set(CMAKE_INSTALL_RPATH_USE_LINK_PATH TRUE) include(
+                        GNUInstallDirs)
+#Define RPATH for executables via a relative expression to enable a
+#fully relocatable package
+                        file(RELATIVE_PATH relDir1 ${CMAKE_CURRENT_BINARY_DIR} /
+                             ${CMAKE_INSTALL_BINDIR} ${
+                                 CMAKE_CURRENT_BINARY_DIR} /
+                             ${CMAKE_INSTALL_LIBDIR})
+                            file(
+                                RELATIVE_PATH relDir2 ${
+                                    CMAKE_CURRENT_BINARY_DIR} /
```

Comment:
I think something has gone wrong with the formatting here :/

Reply:
Oops! Fixed with force-push.